### PR TITLE
Create test.c

### PR DESCRIPTION
**Pull request by minsang-alt**
1931286_김민상

### CODE REVIEW

File: 1931286_김민상/test.c [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+using namespace std;
+
+int main()
+{
+    cout << test <<'n';
+}
```
Comment (ois0886) [inline 1931286_김민상/test.c:7]: 굿

**Comment by minsang-alt** [comment]
LGTM

**Review by acceptor-gyu — APPROVED** [review]
LGTM

**Review by ois0886 — APPROVED** [review]
LGTM

**Pull request merged**